cpp/tests/strings/case_tests.cpp

```diff
@@ -149,6 +149,30 @@ TEST_F(StringsCaseTest, Title)
   CUDF_TEST_EXPECT_COLUMNS_EQUAL(*results, expected2);
 }
 
+TEST_F(StringsCaseTest, IsTitle)
+{
+  cudf::test::strings_column_wrapper input({"Sⱥⱥnich",
+                                            "Examples Abc",
+                                            "Thesé Strings",
+                                            "",
+                                            "Are The",
+                                            "Tést strings",
+                                            "",
+                                            "N2Vidia Corp",
+                                            "SNAKE",
+                                            "!Abc",
+                                            " Eagle",
+                                            "A Test",
+                                            "one More"},
+                                           {1, 1, 1, 0, 1, 1, 1, 1, 1, 1, 1, 1, 1});
+
+  auto results = cudf::strings::is_title(cudf::strings_column_view(input));
+
+  cudf::test::fixed_width_column_wrapper<bool> expected({1, 1, 1, 0, 1, 0, 0, 1, 0, 1, 1, 1, 1},
+                                                        {1, 1, 1, 0, 1, 1, 1, 1, 1, 1, 1, 1, 1});
```

Comment:
shouldn't `"one More"`  return False 0 here?

Reply:
Thanks, yes! I did not notice the test correctly fails here.